.travis.yml

```diff
@@ -64,7 +64,7 @@ script:
   - './cc-test-reporter sum-coverage coverage/codeclimate.*.json'
   - './cc-test-reporter upload-coverage'
   - 'bundle exec bundle-audit check --update --ignore CVE-2015-9284'
-  # - yarn build-storybook
+  - yarn build-storybook
```

Comment:
Renabling storybook builds in CI

Reply:
@nickytonline FYI: since this is not a test, I'll probably move this to `after_script` after #6645 is merged if that's ok with you?